cmd/kubeadm/app/phases/certs/certs.go

```diff
@@ -545,6 +678,63 @@ func getAltNames(cfg *kubeadmapi.MasterConfiguration) (*certutil.AltNames, error
 			altNames.IPs = append(altNames.IPs, ip)
 		} else if len(validation.IsDNS1123Subdomain(altname)) == 0 {
 			altNames.DNSNames = append(altNames.DNSNames, altname)
+		} else {
+			fmt.Printf("[certificates] WARNING: '%s' was not added to the '%s' SAN, because it is not a valid IP or RFC-1123 compliant DNS entry\n", altname, kubeadmconstants.APIServerCertName)
```

Comment:
nit: might be worth splitting over several lines

Reply:
I'll add line-breaks between the args -- I think this is what you mean 👍

Reply:
also done in ac17b344 -- thanks :)